src/inputs/select/select.jsx

```diff
@@ -88,6 +89,9 @@ const propTypes = {
      * The `id` of the Select.
      */
     id: PropTypes.string,
+
+    isRedacted: PropTypes.bool,
```

Comment:
It'd be nice to put a documentation comment here so it shows up on the UI Docs page.